backend/backend/core/models.py

```diff
@@ -6,11 +6,9 @@ class ModelRegistry(object):
     def __init__(self):
         self.models = {}
         self.ollama = OllamaBackend()
-        self.models["chat"] = self.ollama.get_chat_model()
-        self.models["chat_json"] = self.ollama.get_chat_model_json()
 
     def get_chat_model(self):
-        return self.models["chat"]
+        return self.ollama.get_chat_model()
 
-    def get_chat_model_json(self):
-        return self.models["chat_json"]
+    def get_chat_model_json(self, format="json"):
```

Comment:
The `get_chat_model_json` method now takes an optional format parameter, but it's not clear if this change is intentional or a bug.